### PR TITLE
Ps externaldb doc

### PR DESCRIPTION
**Pull request by psantos1113**
- Update the documentation to provide clear instructions when an external database is used in docker swarm.  
- Add certificate configuration back to the installation README. 

- Add to the helm chart documentation.

### CODE REVIEW

File: deployment/docker-swarm/external-db/README.md [hunk 1]
```diff
@@ -0,0 +1,771 @@
...
+            name: "<STACK_NAME>_ALERT_DB_USERNAME"
+            
+    ```
+    - Replace <STACK_NAME> with the name of the stack to be used in the deployment.
+
+##### 4. Create ALERT_DB_PASSWORD secret.
+
+- Create a docker secret containing the database password for Alert.
+    ```bash
+    docker secret create <STACK_NAME>ALERT_DB_PASSWORD <FILE_CONTAINING_PASSWORD>
```
Comment (gkillough) [inline deployment/docker-swarm/external-db/README.md:150]: Should there be an `_` after `<STACK_NAME>`?

File: deployment/docker-swarm/external-db/README.md [hunk 2]
```diff
@@ -0,0 +1,771 @@
...
+              name: "<STACK_NAME>_ALERT_ENCRYPTION_GLOBAL_SALT"
+            
+    ```
+    - Replace <STACK_NAME> with the name of the stack to be used in the deployment.
+
+##### 3. Create ALERT_DB_USERNAME secret.
+
+- Create a docker secret containing the database username for Alert.
+    ```bash
+    docker secret create <STACK_NAME>ALERT_DB_USERNAME <FILE_CONTAINING_USER_NAME>
```
Comment (gkillough) [inline deployment/docker-swarm/external-db/README.md:124]: Should there be an _ after <STACK_NAME>?

File: deployment/docker-swarm/external-db/README.md [hunk 3]
```diff
@@ -0,0 +1,771 @@
...
```
Comment (gkillough) [inline deployment/docker-swarm/external-db/README.md:274]: Should there be an `_` after `<STACK_NAME>`?

File: deployment/docker-swarm/external-db/README.md [hunk 4]
```diff
@@ -0,0 +1,771 @@
...
+            name: "<STACK_NAME>_ALERT_DB_USERNAME"
+            
+    ```
+    - Replace <STACK_NAME> with the name of the stack to be used in the deployment.
+
+##### 4. Create ALERT_DB_PASSWORD secret.
+
+- Create a docker secret containing the database password for Alert.
+    ```bash
+    docker secret create <STACK_NAME>ALERT_DB_PASSWORD <FILE_CONTAINING_PASSWORD>
```
Comment (gkillough) [inline deployment/docker-swarm/external-db/README.md:325]: Should there be an `_` after `<STACK_NAME>`?

File: deployment/docker-swarm/external-db/README.md (hunk 1 above)
Comment (psantos1113) [reply deployment/docker-swarm/external-db/README.md:150]: Thank you

**Pull request merged**